core/domain/prod_validation_jobs_one_off.py

```diff
@@ -613,28 +613,46 @@ def _get_model_id_regex(cls, unused_item):
         return r'^%s$' % USER_ID_REGEX
 
     @classmethod
-    def _get_exp_ids(cls, unused_item):
+    def _get_exp_ids(cls):
```

Comment:
Weird. Any idea why this was "return []" before?

Reply:
This function was being overriden in the subclasses based on how exploration_ids are provided them. For example, `item.exploration_ids` or `item.completed_explorations`.